packages/zoe/tools/manualTimer.js

```diff
@@ -31,12 +31,12 @@ export default function buildManualTimer(log, startValue = 0n, timeStep = 1n) {
     assert.typeof(delay, 'bigint');
     assert(
       delay % timeStep === 0n,
-      `timer has a resolution of ${timeStep}; ${delay} is not divisible`,
+      X`timer has a resolution of ${timeStep}; ${delay} is not divisible`,
```

Comment:
Just pointing out that this one is also arguable, but I would also choose to redact these time values, as you've done here. So no change suggested.